x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form.tsx

```diff
@@ -118,66 +150,115 @@ export function SloEditForm({ slo }: Props) {
+          <EuiSteps
+            steps={[
+              {
+                title: i18n.translate('xpack.observability.slo.sloEdit.definition.title', {
+                  defaultMessage: 'Define SLI',
+                }),
+                children: <SloEditFormIndicatorSection />,
+                status: isIndicatorSectionValid ? 'complete' : 'incomplete',
+              },
+              {
+                title: i18n.translate('xpack.observability.slo.sloEdit.objectives.title', {
+                  defaultMessage: 'Set objectives',
+                }),
+                children: showObjectiveSection ? <SloEditFormObjectiveSection /> : null,
+                status: showObjectiveSection && isObjectiveSectionValid ? 'complete' : 'incomplete',
+              },
+              {
+                title: i18n.translate('xpack.observability.slo.sloEdit.description.title', {
+                  defaultMessage: 'Describe SLO',
+                }),
+                children: showDescriptionSection ? <SloEditFormDescriptionSection /> : null,
+                status:
+                  showDescriptionSection && isDescriptionSectionValid ? 'complete' : 'incomplete',
+              },
+            ]}
+          />
 
-        <EuiFlexGroup direction="row" gutterSize="s">
-          <EuiButton
-            color="primary"
-            data-test-subj="sloFormSubmitButton"
-            fill
-            isLoading={isCreateSloLoading || isUpdateSloLoading}
-            onClick={handleSubmit}
-          >
-            {isEditMode
-              ? i18n.translate('xpack.observability.slo.sloEdit.editSloButton', {
-                  defaultMessage: 'Update SLO',
-                })
-              : i18n.translate('xpack.observability.slo.sloEdit.createSloButton', {
-                  defaultMessage: 'Create SLO',
-                })}
-          </EuiButton>
-
-          <EuiButton
-            color="ghost"
-            data-test-subj="sloFormCancelButton"
-            fill
-            disabled={isCreateSloLoading || isUpdateSloLoading}
-            onClick={() => navigateToUrl(basePath.prepend(paths.observability.slos))}
-          >
-            {i18n.translate('xpack.observability.slo.sloEdit.cancelButton', {
-              defaultMessage: 'Cancel',
-            })}
-          </EuiButton>
+          <EuiFlexGroup direction="row" gutterSize="s">
+            <EuiCheckbox
+              id="createNewRuleCheckbox"
+              checked={isCreateRuleCheckboxChecked}
+              data-test-subj="createNewRuleCheckbox"
+              label={
+                <span>
+                  {i18n.translate('xpack.observability.slo.sloEdit.createAlert.title', {
+                    defaultMessage: 'Create Alert rule for',
+                  })}{' '}
+                  <strong>
+                    {i18n.translate('xpack.observability.slo.sloEdit.createAlert.ruleName', {
+                      defaultMessage: 'SLO burn rate',
+                    })}
```

Comment:
About the copy, should we lowercase `Alert`? What about something like `Create an SLO burn rate alert rule`. We could link to the documentation when we have one and/or google sre book definition. what do you think?
 
I'm also wondering if we need to tell the customer that the configuration of the rule will happen in a following step? Maybe with a EuiIconTip?

Reply:
@kdelemme I was following @maciejforcone's screenshots. 

We could do `Create an SLO burn rate alert`. The actions menu would look like this:

<img width="278" alt="Screenshot 2023-04-12 at 15 29 46" src="https://user-images.githubusercontent.com/535564/231473220-7dd868bf-c90e-4f95-bc37-ef28188e4781.png">

Wdyt?

Reply:
I was following @maciejforcone design shared in Slack.

I'm fine either way.

I like the idea of having a tooltip, will add that. Do you have a link to the right documentation?

Reply:
Cool, updated!